### PR TITLE
Remove data-panel

### PR DESCRIPTION
**Pull request by LJBabbage**
# What and why?
data-panel-header is no longer used in our html templates or the design system so this is now redundant and can be removed

# How to test?
Double check there is no usage and everything looks as it did before
# Jira

### CODE REVIEW

File: response_operations_ui/static/css/main.css.map [hunk 1]
```diff
@@ -1 +1 @@
...
...\n}\n\n.ci-list {\n    border-collapse: collapse;\n    width: 100%;\n}\n\n.ci-list th,\n.ci-list td {\n    text-align: left;\n    padding: 0.5rem;\n}\n\n.ci-list th {\n    border-bottom: 1px solid #222;\n}\n\n/* Collection exercise - Reporting unit styles */\n\n.ceru-headline {\n    background: #f5f6ff;\n    padding: 1rem;\n    margin: 1rem 0;\n}\n\n.tempHack {\n    width: 2.5rem;\n    display: inline-block;\n}\n\n.error-message {\n    color: #d0021b;\n}\n\n/* Social */\n\n.cosy {\n    display: inline-block;\n}\n\n.case-search {\n    background-color: #e4e8eb;\n}\n\n.case-search-input2 {\n    width: 8rem;\n}\n\n.case-search-submit2 {\n    padding: 0.5rem 1rem;\n    margin-left: 0.5rem;\n}\n\n.case-search-results-info {\n    margin: 1rem 0 2rem 0;\n}\n\n.case-search-results-list {\n    list-style: none;\n    padding-left: 0;\n}\n\n.case-search-results-list__item {\n    margin-bottom: 1rem;\n}\n\n.results-list-ref {\n    margin-right: 1rem;\n}\n\n.results-list-postcode {\n    margin-right: 1rem;\n}\n\n.results-list-survey {\n    display: none;\n}\n\n.results-list-status {\n    display: none;\n}\n\n/* temporary styles for social */\n\n.case-address__prem {\n    display: block;\n}\n"]}
```
Comment (matthew-robinson-ons) [inline response_operations_ui/static/css/main.css.map:1]: There are still some references to the data panel within this file. though not sure how these are generated.

Comment (LJBabbage) [reply response_operations_ui/static/css/main.css.map:1]: interesting, this is the autogenerated file

**Review by matthew-robinson-ons — APPROVED** [review]
This is looking good now and all tests passing

**Review by pricem14pc — APPROVED** [review]
Sanity tested as much functionality as possible. Not seeing any issues or scripts in the browser console related to the data-panel.

**Pull request merged**